### ISSUE TITLE
Doesn't work on IE 11 without polyfil

### ISSUE THREAD

**Issue by acierto** [stefalda/localized-strings#2]
Hi!

Nice lib, but why as a dist not to provide es5 compatible code? 99% of libs does it, so you don't need to include polyfill in your code for that.

**Comment by benkeen** [comment]
Yeah, noticed this too. It whines about `Object.assign()` on IE11.

**Comment by fbacker** [comment]
@acierto and @benkeen can you please try out latest release. Is it fixed?
